Alcatraz/Controllers/ATZPluginWindowController.m

```diff
@@ -89,10 +89,18 @@ - (BOOL)userNotificationCenter:(NSUserNotificationCenter *)center shouldPresentN
 - (IBAction)installPressed:(ATZFillableButton *)button {
     ATZPackage *package = [self.tableViewDelegate tableView:self.tableView objectValueForTableColumn:0 row:[self.tableView rowForView:button]];
     
-    if (package.isInstalled)
-        [self removePackage:package andUpdateControl:button];
-    else
+    if (package.isInstalled) {
+        if ([package isKindOfClass:[ATZPlugin class]] && [(ATZPlugin *)package isBlacklisted]) {
```

Comment:
Agreed. I wondered how to do that elegantly. Would moving `isBlacklisted` up into `ATZPackage` be preferable? Polymorphism would be restored, but then being blacklisted doesn't make much sense for a Template or Color Scheme... (ie. these 2 implementation would just `return NO`)